assets/support.rackspace.com/src/css/less/components/search-box.less

```diff
@@ -0,0 +1,58 @@
+[data-carina-search-box] {
```

Comment:
So, the only thing that script doesn't do is the colour functions. But I think we'd have to comb through manually anyway. It's worth a short on this PR. Thoughts?